cloudman/helmsman/management/commands/helmsman_load_config.py

```diff
@@ -24,7 +24,8 @@ def process_settings(settings):
             print("An exception occurred during helm init:", str(e))
         for repo in settings.get('repositories'):
             call_command("add_repo", repo.get('name'), repo.get('url'))
-        for chart in settings.get('charts'):
+        for chartKey in settings.get('charts'):
```

Comment:
I think this can be a one liner and avoid the lookup with:
```suggestion
        for chart in settings.get('charts', {}).values():
```